### ISSUE TITLE
Local clones make and missing clone of extracted gem `build install` fail

### ISSUE THREAD

**Issue by dg-ratiodata** [cucumber/cucumber-ruby#961]
## Summary

Extracted wire-protocol-gem makes `bundle install` fail.

## Expected Behavior

~~~
bundle install
~~~

should succeed.

## Current Behavior

~~~
bundle install
~~~

fails with 

~~~
The path `/home/user/data/projects/cucumber-ruby-wire` does not exist.
~~~

## Possible Solution

Since I opened another PR #960 a few minutes ago which affects the same lines
but because of a different thing, I just openened an issue.

~~~ruby
gem "cucumber-pro", "0.0.13", :group => :test
source "https://rubygems.org"
gemspec
unless ENV['CUCUMBER_USE_RELEASED_CORE']
  core_path = File.expand_path("../../cucumber-ruby-core", __FILE__)
  wire_path = File.expand_path("../../cucumber-ruby-wire", __FILE__)

  if File.exist?(core_path) && !ENV['CUCUMBER_USE_GIT_CORE']
    gem 'cucumber-core', :path => core_path
  else
    gem 'cucumber-core', :git => "git://github.com/cucumber/cucumber-ruby-core.git"
  end

  if File.exist?(wire_path) && !ENV['CUCUMBER_USE_GIT_CORE']
    gem 'cucumber-wire', :path => wire_path
  else
    gem 'cucumber-wire', :git => "git://github.com/cucumber/cucumber-ruby-wire.git"
  end
end
~~~

Additionally we could add a bootstrap-script. Something really simple:

script/bootstrap
~~~
#!/usr/bin/env bash

git clone https://github.com/cucumber/cucumber-ruby
git clone https://github.com/cucumber/cucumber-ruby-core
git clone https://github.com/cucumber/cucumber-ruby-wire
~~~

This might be even useful for other cucumber projects. Maybe something similar like [Github](http://githubengineering.com/scripts-to-rule-them-all/) does.

## Steps to Reproduce (for bugs)

1. `git clone https://github.com/cucumber/cucumber-ruby`
2. `git clone https://github.com/cucumber/cucumber-ruby-core`
3. `cd cucumber-ruby`
4. `bundle install`

## Context

I had `cucumber-ruby-*` locally before the `wire`-protocol was extracted to a separate gem.

## Your Environment

* Version used: master
* Operating System and version: Not relevant in this case
* Link to your project: Not relevant in this case

**Comment by danascheider** [comment]
I was able to repro this on my machine. I'm going to take a look at the code, @dg-ratiodata - I don't really think this is an acceptable state of affairs so let's get it fixed!

**Comment by mattwynne** [comment]
I think the changes suggested to the Gemfile are the way to go - we'll hopefully never need to touch the wire protocol code again, so there's not much point in making the bootstrap script to make it easy to hack on.

**Comment by dg-ratiodata** [comment]
Thanks a lot.

**Comment by danascheider** [comment]
Closing this since I merged #983
